File: 백준/Silver/1929. 소수 구하기/소수 구하기.py
def prime(n):
    if n<2: return 0
    for i in range(2, int(n**0.5)+1):
        if n%i==0: return 0
    return 1
start, end = map(int, input().split())

for i in range(start, end+1):
    if prime(i): print(i)

File: 백준/Silver/1929. 소수 구하기/소수 구하기.cc
#include <cstdio>

int prime(int n) {
    if (n<2) return 0;
    for (int i=2;i*i<=n;i++) {
        if (n%i==0) return 0;
    }
    return 1;
}
int main(void) {
    int start, end; scanf("%d %d", &start, &end);
    for (int i=start; i<=end; i++) {
        if (prime(i)) printf("%d\n", i);
    }
    return 0;
}